Assets/Atlas games/Scripts/ScreenShake.cs

```diff
@@ -0,0 +1,48 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Rendering;
+
+public class ScreenShake : MonoBehaviour
+{
+
+    public static ScreenShake instance;
+
+    private float shakeTimeRemaining, shakePower, shakeFadeTime, shakeRotation;
+
+    public float rotationMultiplier = 15f;
+
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        instance = this;
+    }
+
+    private void LateUpdate() {
+        if(shakeTimeRemaining > 0f)
+        {
+            shakeTimeRemaining -= Time.deltaTime;
+
+            float xAmount = Random.Range(-1f, 1f) * shakePower;
+            float yAmount = Random.Range(-1f, 1f) * shakePower;
+
+            transform.position += new Vector3(xAmount, yAmount, 0);
+
+            shakePower = Mathf.MoveTowards(shakePower, 0f, shakeFadeTime * Time.deltaTime);
+
+            shakeRotation = Mathf.MoveTowards(shakeRotation, 0f, shakeFadeTime * rotationMultiplier * Time.deltaTime);
+        }
+
+        transform.rotation = Quaternion.Euler(0, 0, shakeRotation * Random.Range(-1f, 1f));
```

Comment:
instead of LateUpdate use Coroutine. 
Don't block the main thread with something that is not needed all the time.

Reply:
OK, I check it and send it to you tomorrow.